llama_stack/providers/inline/post_training/huggingface/config.py

```diff
@@ -71,7 +71,7 @@ class HuggingFacePostTrainingConfig(BaseModel):
     dpo_beta: float = 0.1
     use_reference_model: bool = True
     dpo_loss_type: Literal["sigmoid", "hinge", "ipo", "kto_pair"] = "sigmoid"
-    dpo_output_dir: str
+    dpo_output_dir: str = ""
```

Comment:
```suggestion
    dpo_output_dir: str | None = None
```

I think None is a more suitable default rather than an empty string, some checks for `None` would be great too in the code, thanks!